test/usecases/log/GetAllLogsUseCase.test.js

```diff
@@ -43,6 +43,19 @@ module.exports = () => {
         }
     });
 
+    it('should return logs with a full tag collection regardless of filter', async () => {
```

Comment:
note to self: This test should probably work the other way around (filtered result first, then compare against unfiltered)